backend/app/views/spree/admin/orders/index.html.erb

```diff
@@ -157,7 +157,7 @@
   </table>
 <% else %>
   <div class="alpha twelve columns no-objects-found">
-    <%= Spree.t(:no_resource_found, resource: I18n.t(:other, scope: 'activerecord.models.spree/order')) %>,
+    <%= Spree.t(:no_resource_found, resource: Spree::Order.model_name.human(count: :other))  %>,
```

Comment:
This one here needs to be changed. This is perfect example for setting grammatical order in views and nobody can translated this into correct grammar of the language. The comma, the bang. Everything has to go into one translation key with interpolation variables. Everything else is just pretending I18n